tests/utility/v2/test_component_analyses.py

```diff
@@ -46,6 +55,17 @@ def test_known_package_flow(self, _mock1, _mock2):
         result = known_package_flow('pypi', "django", "1.1")
         self.assertTrue(result)
 
+    @patch('bayesian.utility.v2.component_analyses.g')
+    @patch('bayesian.utility.v2.component_analyses.server_create_component_bookkeeping')
+    def test_get_known_unknown_pkgs_no_cve(self, _mock1, _mock2):
+        """Test Known Unknown Pkgs, No Cve."""
+        normalised_input_pkgs = [('markdown2', "2.3.2")]
+        stack_recommendation, unknown_pkgs = get_known_unknown_pkgs(
+            "pypi", self.gremlin_batch_data_no_cve, normalised_input_pkgs)
+        ideal_output = [{'package': 'markdown2', 'version': '2.3.2', 'recommendation': {}}]
+        self.assertListEqual(stack_recommendation, ideal_output)
+        self.assertSetEqual(unknown_pkgs, set())
+
 
```

Comment:
I guess you may need to add another test to cover a case 1 pkg with vuln & another without vuln?

Reply:
Ack!